### DISCUSSION TITLE
qmk_firmware ver.0.22.3環境でビルドしたファームウェアでトラックボールが動作しない

### DISCUSSION THREAD

**Discussion by terwe52431** [mentako-ya/mtk#21] [Q&A] [answered]
qmk_firmware ver.0.22.3、両側ボール環境でビルドしたファームウェアでポインターが動かずスクロールもしません。

config.hの設定は以下の通りでその他はチェックアウト時の初期状態のままです。
// #### 自動マウスレイヤー切替有効化 ####
#define AUTO_MOUSE_LAYER_ENABLE

// #### エンコーダー設定 ####
// 左エンコーダー有効 左ボール無効（デフォルト）
#define ENCODER_LEFT_ENABLE
// 右エンコーダー有効 右ボール無効
//#define ENCODER_RIGHT_ENABEL

// #### 両側ボール設定 ####
#undef ENCODER_ENABLE
// slave側トラックボールをスクロール操作に使用する場合（デフォルト）
#define SLAVE_SCRL_DISABLE 0
// slave側トラックボールをポインター操作に使用する場合
//#define SLAVE_SCRL_DISABLE 1

アップロードされていたmtk_mtk64e_PMW3360_AutoMouseLayer_via_db.0.22を書き込めばポインターが動くことは確認しています。
また、qmk_firmware ver.0.16環境でこちらでビルドしたファームウェアでもポインターは動くことは確認しています。

**Comment by mentako-ya** [comment, marked as answer]
// 左エンコーダー有効 左ボール無効（デフォルト）
#define ENCODER_LEFT_ENABLE
をコメントアウトしてみてください
